tests/bluetooth/mesh_shell/src/main.c

```diff
@@ -18,11 +18,64 @@ static struct bt_mesh_cfg_cli cfg_cli;
 
 BT_MESH_SHELL_HEALTH_PUB_DEFINE(health_pub);
 
+static void show_faults(uint8_t test_id, uint16_t cid, uint8_t *faults, size_t fault_count)
```

Comment:
Could `bt_mesh_shell_health_cli` be still used here?

Reply:
When building the test application I get deprecation warning when applying the old health cli instance. Should I remove deprecation tag in shell.h then?

Reply:
Yes